<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>581</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="selectApp">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>190</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLanguage">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Lengua</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="appNameExe">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>301</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSpeech">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Grabacion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="speechName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>301</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelApp">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Aplicacion</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>301</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Maya</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Yaqui</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Nahuatl</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="selectSpeech">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelIcon">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Icono</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="appIcon">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>230</y>
       <width>301</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="selectIcon">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>230</y>
       <width>99</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAppName">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="appName">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>301</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelAppDescription">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Descripcion</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
